Automattic/vip
# FORNO-1512: Add prompt for selecting a backup

## Description

This PR prompts the user to select a backup for them to `sync` (will work for `export` soon!)

Also adds the skeleton for a future feature

## TODO

- [ ] Unit tests for functions that are in use.
- [ ] Nice to have: A separate PR for fixing all the typescript issues (this PR solves some but not all)

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Run `npm run build`
1. Run `node dist/bin/vip-dev-env-sync-sql.js --slug=0000-production @0000-production`
1. You should get a prompt to import some backups and not just the latest :)
1. Everything should just work.
